cmd/ctr/commands/images/import.go

```diff
@@ -122,6 +122,18 @@ If foobar.tar contains an OCI ref named "latest" and anonymous ref "sha256:deadb
 				return err
 			}
 		}
+
+		ctx, done, err := client.WithLease(ctx)
+		if err != nil {
+			return err
+		}
+		defer func() {
```

Comment:
simply `defer done(ctx)` is enough here and what is done in most places. If lease cleanup fails due to connection it will still get cleaned up later in the GC, if it fails in the backend the errors there will be more useful. Its not expected that a CLI user must act on a lease deletion failure though.

Reply:
I see. Will update.